docs/change-log/index.md

```diff
@@ -13,16 +14,181 @@
 - [Limitations](#limitations)
 - [Deprecations](#deprecations)
 
+## v1.3.0
+
+### Client/server matrix
+
+
+
+- The base images mlrun/mlrun:1.3.0 etc. are based on python 3.9. <br>
```

Comment:
I guess this is under the `Client/server matrix` section due to the fact that the MLRun server is now 3.9-based (which is not written here, this just mentions the mlrun images), and we recommend moving the client to 3.9 as well. This is what these steps are for, and I think it makes sense to explain it.